web/src/app/core/services/monitor.service.ts

```diff
@@ -65,6 +74,12 @@ export class MonitorService {
       );
   }
 
+  /**
+   * Delete the monitor from project
+   * @param projectUid uid of project
+   * @param monitorUid uid of monitor to be deleted
+   * @returns the observable
```

Comment:
Not consistent with the `returns` below, this is a description and below is the `type`.

Many places do not the return, we need to always be consistent.

I think we discussed to remove the `returns` as the documentation adds it automatically? Is the correct?

Reply:
removed the @returns from all places.